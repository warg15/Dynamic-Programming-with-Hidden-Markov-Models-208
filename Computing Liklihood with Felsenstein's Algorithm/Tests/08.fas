>Rifleman
GGATCGGATCCTCGTTTACAAGCTTTGCACCCCTTAAACAACAAATCTGATGTATTTATCGATGGTAAGGCCCATTCCCTCTTAAGTTATTTTAGAATAACTCTAAAGAAGAGTCTGGTACCAGAGATTTTGTAGATACTAATATAGGGATGTTAAGGCTCCGGGGACCTGTGGGAAAGACTTGGGTAACTGATGCTAGC
>Chimney-swift
CGATTGAATCCTCGGTTGCAGGCTTGGCACCCCTAAGACAATAAATCTGATGTTTTTATCGATAGTAAGGGGCATTACCTCTTAAATCACTTTATAATAACGGTAAAAAAAAATCTGGCACCAGAGATTTTGTATATACTAATATAGGGATGTTAAGGCCCCGGGGACCTGTGGGTAAGGCTTGGGTAACCGATCCTAGC
>Yellow-throated-sandgrouse
GGATCGAATCCTAGGTTGCAGGCTTGGCAGCCTTGGAACAACAAGTCTGATGCTTTTATCTATAGTAAGGCCCATTACCTCTCTAGTTACTTTGGACTAACTCTAAAGTAGAGTCTGGCACCAGAGGTCTTGTGAGTACTAATATAGGGGTGTTAAGGCCCCGGGGACTTGTGGGGGAGACTTGGGTAACTGATGCTAGC
>Downy-woodpecker
GGATCGAAGCCTCGGTTACAAGCTTGCCAGCCCTAAAACAAAAAATCTGATGTTTTTACCGATAGAACGGCCCATGCCCTCTTCAGATACTTTAGAGTAACTTTAAAGGAAGGTCTTGCACCAAAGATTTTGTCGATACTAACATATGGGTGTTAAAGCTCCGGGGGCCTGTGGGGAAGATTAGGGTAATTGAGCCTAAG
>Northern-fulmar
GGATCGAATCCTCGGTTACAAGCTTGGCAGCACTAAAACAACAAATCTGATGTTTTTATCGATAGTAAGGCCCACTCCCTCTTTAGTTACTTTAGAATAGCTCCAAAGAAGCGTCTGGCACCAGAGATTTTGTGGATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTATGGGGAAGACTTGGGTAACTGATCCTAGT
>Common-ostrich
AGATCGACTCATCGGTTTCAAGCTTGGCGGCCCTAAAACAACAAATCTGATGTTCTTATCGATAGTTAGGCCCATTACCTCTTCAATCACTTCAGAATAACTCTAAAGAAGAGTCTGACAACAGAAATTTTATTGGTACTAATATAGGTGAGTTAAAGCCCCAGGCATCTTCGGGGAAAGTTTAGGCGACTAAGCCTAGC
>Red-throated-loon
GGATCGAATCCTCCGTTGCAAGCTTGGTAGCCCTAAAACAACAAATCTGATGTTTTTATCGATAGTAAGGCCCATTCCCTCTTTAGTTACTTTAGAATAACTCTAAAGTAGAGCCTGGCACCAGAGATTTTGTGGGTACTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATTCTAGC
>Great-cormorant
GGATCGAATCCTCGGTTGCAAGCTTGGCAGTCCTAAAACAACAAATCTGATGTTTTCATCGATAGTAAAGCCCAGTCCCTCTTTAGTTACTTTAAAATAACTCTAAAGAAGAGTCTGGCACCAGAGATTTTGTGGACACCAATATAGGGGTGTTAAGGTCCCAGGGATCTGTGGGGAAGACTTGGGTAACTGATCCTAGC
>Grey-crowned-crane
GGATCGAATCCTCGGCTGCAGGCTTGGCAGCCCTAAAGCAACAAATCTGACGCTTTTATCGATAGCAAGGCCCATTCCCTCTTCAGTTACTTTATAATAACTCTAAAGAGGAGTCTGGCACCAGAGATTTTGTGGATACTAATATAGGGGTGTTAAGGCCCCGGAGACCTGTGGGGAAGGCCTGGGTAACTGATCCTAGC
>Kea
GGCTCAAATCCTCGGTTGCAAGCTTGGCAGCCCTAAAACAACAAACTTGATGCTTTTATCTATGGTAAGGCCCATTGCCTCTTCAATTACATTAGAATAACTCTATAGAAGAGTCTGGCGCCAAAGATTTTGCGGATACTGATATAGGGGTGTTAAGGCCCCTGGGACCTCTGGGCAAGACTTGGGTAACTGATCCTAGC
>Golden-collared-manakin
GGATCGGGTCCTCGTTTACAAGCTTTACACCCCTTAGACAACAAATCTGATGTTTTTATATATAGTAACGCCCATTCCCTCTTAAGTTATTTTACAATAATTCTAAAGAAGAGTCTGGCACCAAAGACTTTGTAGATACTAATATAGTGGTGTTAAGGCTGCGGGGAGCAGTGGGAAAGACTTGGGTAACTGATCCTAGC
>Brown-mesite
GGATCGAATCCTCTGTTGCAAGCTTGGCAGCCTTAAAACAACAAATCTGATGTTTCTATCGATAATAAGGCCCATTATCTCCTCAGTTATTTAAGAATAATTCTAAAGAAAAGTCTGGCAGCAGAGATTTTGTAGATACTAATATAGGGGTGCTAAGTCCCCGGGGACCTGTGGGGAAGATTTGGGTAACTGATCCTAGC
>White-throated-tinamou
AGGTCGATTCATCGGTTTAAAGCTCGGTGGCTTTAAAACAATAAAGCTGATATTCATATCCGTACTGATACCCATTACCTCTTCAATCACTTCATAATAACTCTAACGCAGAGTCTGACAACAGAAATCTTATTAGTACTAATATAGGGGGGTTAACGCCCCAGGCACCTCTGGAGAAAACTTGGGAGACTGATCCTAGC
>Pigeon
GGATTGGATCCTCGGTTGCAACCTTGGCAGCCCTAAAACAACAAATCTGATGTTTTTATCGGTAGCAAGGCCCACTACTTCTTCAGTTACTTAAGAACAACTCTAAAGAAGAGTCTGGCATCAGAGTTTTTGTGGATACTAATATAGGGTTATTAAGGCCCCGGGGACCTGTGGGCAAGACTTGGCTAACTGATGCTAGC
>White-tailed-eagle
GGATCGAATCCTCGGTTGAAAGCTTGGAAGCCCTAAAACAACAAATCTGATGTTTTTATCGATAGTAAGGCCCATCCCCTCTTCAGTTACTTTAGAATAACTCTAAAGAAGAGTCTGGCACCAGAGATCTTATGGATACTGATACAGGGGTGTTAAGGCCCCGGGGACCTCTGGGGAAGACTTGGGTAACTGGTCCTAGC
>Sunbittern
GGATCGAATCCTAAGTTGCAGGCTTGGCGGCACTAAAGCGACAAATTTGATGTTTTTATTGATGGTACGGCCCATTCCCTCTTTAGTTACTTTATAATAACTCTCAAGAAGAGACTGGCACCAGAGATTTTGTGGATACTAATATAGGGATGTTAAGATCCCGAGAACCTGTGGGGAAGACTTGGGTAACTTATCCTAGC
>Dalmatian-pelican
GGATCGAATCCTTGGTTGCAAGCTTGGCGGCCCTAAAACAACAAATCTGATGCTTTTATTGATAGTAAGGCCCATTCCCTCTTTAGTTACTTTAGAATAACGCCAAGGAAGAGTCTGGCACCAGAGACTTTGTGGATACTGATATAGGGGTCTTAAGGCGCCAGGGACCTGTGGGGAAGATTTGGGTAACTGATCCTAGC
>Zebra-finch
GGAACGGATCCTTGTTTACAAGCTCTGCACCCTTTAGACAACAAAGCTGATGTTTTTATCGATAGTAAGGCCAATTTCCTCTTAAGTTTTTTTAGAATAACTCTTAAGAAGAGTCTGGCACCAGAGATTTTGTAGATACTAAAATAGGGGTGTTACGGCTGCGAGGACCTGTGGGAAAGACTTGGGCAACTGATCCTAGC
>White-tailed-tropicbird
GGATCGAATCCTCGGTTGCAAGCTTGGTAGCCCTAAAACACTAAATCTGATGTTTTTATTGACAGTAAGGCCCATTCCCTCTCCAGTTACTTCAGAATAACTCTAAAGAAGAGTCTGGCATCATAGATTTTGTGGATACTAGTATAGGGGTGTTAAGGCCCCGGGGACCTATAGGGAAGACTTGGGTAACTGATCCTAGC
>Speckled-mousebird
GGATCGAATTCTCGGTTGCAAGCTTGGCAGGCCTAAAAAAGTAAAGCTGATGTTTTTATCGGTAGTAAGGTCCATTCCCTCTTCAGTTACATTGGAAGAACTTTAAAGAACAGTCTGGCTCCAAAAATTTTGTGGACATTAAAATAGGGGTGTTAACTCCCCGGGGACCTGTAGGCAAGACTTAGGTAACTGACCTTAGC
>Chuck-wills-widow
GGATCGAATCCTCGGTTGCAAGCTTGGCAGCCCTAGAATAACAAATCTGATGCTTTTATCGATAGTAAGGCCCATTACTCCCTCAGTTACTTTACAATAACTGTAGAAAAGAGTCCGGCACGAGAGATTTTGTGGATACTAATATATGTGTGTTAAGGCTCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATCCTAGC
>Bald-eagle
GGATCGAATCCTCGGTTGAAAGCTTGGAAGCCCTAAAACAACAAATCTGATGTTTTTATCGATAGTAAGGCCCATCCCCTCTTCAGTTACTTTAGAATAACTCTAAAGAAGAGTCTGGCACCAGAGATCTTATGGATACTGATACAGGGGTGTTAAGGCCCCGGGGACCTCTGGGGAAGACTTGGGTAACTGGTCCTAGC
>Adelie-penguin
GGATCGAATCCTCGGTTGCAAGCTTGGCAGCCCTAAAACAACAAATCTGATGTTTTTATCGATGGTAAGGCCCATTCCCTCTTTAGCTATTATAGAATAACTCTAAAGAATAGTCTGGCACCAGAGATTTTGTGGATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATCCTAGC
>Bar-tailed-trogon
GGAGCGAATCCTCGGGTGCAAGCTTGCCAGCCCTGGAACAATAAATCTGATGTTTTTAACGATAGTAAGGCCCATTCCGTCTTCAGTTACTTGAAGACAACTCTACAGAAGATTCTGGCACGAGATATTCTGTGGATACTAATATAGGGGTGTTAAGGCCCCAGGGACCTGTGGGGAAGACCTGCGTAACGAACCCTAGC
>Little-egret
GGATGGAATCCTCGGTTGTAAGCTGGACGGCCCTAAAACAACAAATCTGATGCTTCTATCGATAGTAAGGCCCACTCCCTCTTTAGTTACTTTAGACTACATCTAAAGAAAAGTCTGGCACGAGAGACTTTGTGGATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACCGATCCAAGC
>Common-cuckoo
GCATCGAATCCTCGGTTGCAAGCTTGGAAGTTCTAAAGCACCAAATCTCATGTCTTTCTTGATAGTCATGTTCACTATCTATTCGGTTACTTTAGGTTAACTCTAGAAAAGAGTCTGGCACCAGAGATTTTGTGGATATCAATATAGGGGTGTTGAGGTCCCTGGGACCTGTAGTAAAGACTCGGGTAACTGATCCTAGC
>Turkey
GGATACACTCCCTAGAAGCAAATTTAGCAGCTCTAAAATAATAAATCAGATGTTTTTATAGATAGTTGGGCCCATTACCTTTTCAGTCACTCTGGAATAACTCTATAGAAGAGTCCGGCACCAACAATTTTGTCGATACTAATATAGGCAAGTTAAGGCCCCGGGCACCCATGGGGTAGACTTGGGTTACAAATCCTCGC
>Pekin-duck
GGATCTACTCCTTGGTTGCAAGCTTAGAAACCCTAAAATTACAAATCTTATTTTTTTATAGATAGCGGGGCCCATTACCTCTTCAATCACTTCAGAAAAATTCTAAAGAAGAGTCTTGCACCAGAAAATTTGTCGCAACTAATATAAGTAAACTAAGGCCCCGGGTACCTATGGGGAAGACTTGGGTAACAGAGCCTAGC
>Medium-ground-finch
GGATCGGATCCTTGTTTACAAGCTCTGCACCCGTTAGACAACAAATCTGATCTTTTTATCGATAGTAAGGCCAATTCCCTCTTGAGTTTTTTTAGAATAACTCTAAAGAAGAGTCTGGCACCAGAGATCTTGTAGATACTAATATAGGGGTGTTACGGCTGCGCGGACCTGTGGGAAAGACTTGGACAACTGATCCTAGC
>Rhinoceros-hornbill
GGATCGAATCCTCGGTTGCAAGCTTGCCATCCCTAAAACAACAAATCTGATGTTTTTATCGATAATAGGGCCCATTCCCTCGTCAGTTACTTTAGAATAACTCTAAAGAAGAATCTGGCCTCAGAGATTTTGTGGATACTAATATAGGGGTGTAAACGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATCCTAGC
>American-flamingo
GGATCGAATCCTCGGTTGCAAGCTTGGCAGCCCTAAAACAACAAATCTGATGTTTTTATCGATAGTAAGGCCCATTACCTCTTCAGTTACTTTAGAATAACTCTAAGGAAGAGTCTGGCGCCAGAGATTTTGTGGATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTTTGGTAACTAATCCTAGC
>Peregrine-falcon
GGATCGAATCCTCGGTTGCAAACTTAGCAGCCCTAAAACAACAAATCTGATGCTTTTATCGATAGTAAAGCCCATCCTCTCTTCAGCTACTTTAGAATAAATCTGAAGAAGAGTCTGGCACCAGAGACTTTGTGCATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGAAAGACTTGGGTAAGTGATCCTAAT
>Emperor-penguin
GGATCGAATCCTCGGTTGCAAGCTTGGCAGCCCTAAAACAACAAATCTGATGTTTTTATCGATGGTAAGGCCCATTCCCTCTTTAGTTATTATAGAATAACTCTAAAGAATAGTCTGGCACCAGAGATTTTGTGGATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGGTCCTAGC
>Turkey-vulture
GGATCGAATCCTCGGTTGCAAGCTAGGCAGTCCTAAAACCACAAATCTGATGTTTTTATCGATAGTAAGGCCCATTCACTCTTCAGTTACTTTAGAATAACTCTAAAGAAGAGTCTGGCACCAGAGATTTTGTGGATACTAATAAAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGTAGACTTGGGTAACTGATCCTAGC
>Great-crested-grebe
GGATCGAATCCTCGGTTGCAAGCTTGGCAGCCCTCAAACAAGAAATCTGATGTTTTTATCGATAGTGAGGCCCATTAACTCTTCAGTTACTTTAGAATAACTCTAAGGAAGAGTATTGCGCCAGAGACTTTGTGGATACTAATATAAGGGTGTTAAGGCCCCGGGGACCTTTGGGGAAGACTTTGGTAACTACTCCTAGT
>MacQueens-bustard
GGACGGGATCCTCAGTTGCAAGATTGACAGCCCTAAAACAACGAATCTAATGTTTTTATCGATGGTAAAGCCCATTACCTCTTCATTTAGTTTAGAATAACTCTAAAGAAGAGTCTGGCATCAGAGATTTTGTGGAGACTAATATAGCGGTGTTAAGGCTCCGAGGATCTGTGGGGAAGACTTGCGTAACTGATCCGGGC
>Crested-ibis
GGATCGAATCCTCGGTTGCAAGCTTGCCGGCCCTAATACAACAAATCTGATGCTTTTATTGATAGTAAGGCCCATTCCCTCTTTAGTTACCTTAGAATAACTCTAAAGAAGAGTCTGGCACCAGAGACTTTGTGGATACTACTATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATCCTAGG
>American-crow
GGATCGGATCCTTGTTTACAAGCTTTGCCCCCCTTAAACACCAAATCTGATATTTTTATCGATAGTAAGACCGATTCCCTCTTGAGTTTTTTTAGAATAGCTCTAAAGAAGAGTCTGGCACCAGAGATTTTGTGGATACTAATATAGGGGTGTTAAGGCTGCGGGGACCTGTGGGAAAGACTTGGGCAACTGATCCTAGC
>Carmine-bee-eater
GGATTGAAGCCTCGATTGCAAGCTTGCCTGCCCTAGAACAACAAACCTGATGTTTTAAGCGATAGTAAGGCCCATGCTGTCTTCAGTTACTCTAGAATAACTCTAATGAAGACTATGGCTTCTGAGATTTTGTGGATGCTAATATAGGGGTGTTAAGGCCCCGGGGACCTGTGGGGAAGACTGGGGTAACTAATCCCAGC
>Annas-hummingbird
TGATCGAAGCCTTGATTGCATATTTGGCAGTCCTAAAACAACAAACCTGATGTCTTTATAGATACTAAGAGCCATTACCTCTTAAGTTACGTTACAATAGCTGTGAAGAAGAGTCTGGCCCGAGAGATTTTGTAGATACTAATATAGGAGTATTAAGACCCCAGGGATCTGTGTGGAGGGATTGGGTGACTGATTCTATC
>Barn-owl
GGATCGAATCCTCGGTAGCAAGCTTGGCAGCCTTAAAACAACAAATATGATGTATTCATCGATAGTAAGGCCCATTCCCTCTTCAGTTACTTTAGAATAACTCTAAGGAAAAGTCAGGCACCAGAGATTTTGTGGATACTAATATTGGGGTGTTAAGAACCCGGGGACTTGTAGGGAAGACTTGGGTAACTGATCCTAGC
>Red-crested-turaco
GGATCGAATCCTCGATTGCAAGCCTGGCAGCTCTAAAACAACAAATCTGATGTTTTTATCGAGATTAAGGCCCACTACCTATTCAGTTACTTTAGAATATCTCTAAAGGAGAGTCTGGCACCAGAGATTTTGTGGATACTAAAACAGGGGTACTAAGGCCCCGGGGACATGTGAGGAAGACTTGGGTAACGGATCCTAGC
>Budgerigar
GGCTCGAATCCTCGGTTGCAAGCTTGGCAGCCCTAAACCAACAAACCTGATGTTTTTATAGATGGTAAGGCTCATTGCCTCTTCAATTACTTTAGGATAACTCCATAGAAGAGGCTGGCGTAAAGAATTTTGCGGATACTAATATAGGGGTGTTAAGGCCCCGGGGACCTCTGGGGAAGACTTGGGTAACTGATCCCAGC
>Red-legged-seriema
GGATAGAATCCTCGGTTGCAATCTTGGCAGCCCTAAAACAACAAATCTGATGTTTTTATCAATAGTAAAGCCCACTCCCTGTTCAGTTATTTTAGAATAACTCTAAAGAAGAGTCTGGCACCAGAGATTTTGTGGATACTAATATAGGGGTGTTGAGGCCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATCCTAGC
>Chicken
GAATACAGTCCTTGGAAGCCAACTTAGAACCTCTAAAATAATAAATCTGATGTTTTTATAGATAACAGGGCCCATTACCGTTTCAGTCACTTCGGAATAATTCTATAGAAGAGTCCGGCACCAACAATTTTATCGAGACTGATATAGGCAAGTTACGGCCCCGGGCACCCGTGGGGTAGCCTAGGGTAACACATGCTCGC
>Killdeer
GGATCGAATCCTCGGTTGCAAGCTTGGCGGCCCTAAAACAACAAATCTGACGTTCTTATCGATAGTAAGGCCCATTCCCTCTTCAGTTAATTTAGTATAACTCTAAAGAAGAGTCTGGCACCAGAGATGTTGTGGATACTAATACAGGGGTGTTAAGGTCCCGGGGACCTGTGGGGAAGACTTGGGTAACTGATCCTAGC
>Cuckoo-roller
GGATCAAATCCTCGGTTGCAAGCTTGGTAGCCCAAAAACAACAAATCTGATGTTTTTATTGATAGTAAGACCCATTCCCTCTTCTGTTACTTTAGAATAACTCTAAAGAGGTGTCTGGCACCAGAGATTTTGTGGATACTAATATCGGGGTGCTAAGGCCCCGAGGACCTGTGGGGAAGACTTGGGCAACTGATCCCAGC
>Hoatzin
GAATCGAATCCTCGGTTGCAAGCTCGGCAGCCCTACAACAACAAACCTGACGTTTTTATCGATATTCAGGCCCATTCCCTCTTCAGTTACTTTAGAATAACTCTAGAGAAGAGTCTGGCACCAGAGATTTTGTCGATCCTAATATAGGGGTGTTAAGGCCCCGCGGACCTGTGGGGAAGACTTGGGTAACTGATCCTACA
