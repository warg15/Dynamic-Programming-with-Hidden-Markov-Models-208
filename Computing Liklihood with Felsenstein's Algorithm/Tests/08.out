-3562.5952267803145
